kernel/dev/tty/tty.cpp

```diff
@@ -46,14 +46,14 @@ void px_tty_clear(px_tty_vga_color fore, px_tty_vga_color back) {
     reset_fore = fore;
     reset_back = back;
     volatile uint16_t* where;
-    uint16_t attrib = VGA_COLOR(color_back, color_fore);
+    uint16_t attrib = (uint16_t)VGA_COLOR(color_back, color_fore);
```

Comment:
Can this cast be moved into the macro?

Reply:
I believe I tried that and it didn't work. The reason is that because the macro does operations (mathematical or bitwise) GCC converts it into an `int` (for some reason) so we have to cast the final result and not the individual parameters.

Reply:
Macros are just text replacement, so it should be lexically identical to move the cast inside the macro

Reply:
Oh, I see what you're saying. My only thing with that is it would force the macro to only be used with `uint16_t` types. Is that something we're okay with?